src/Utils/resizer.ts

```diff
@@ -30,9 +30,12 @@ const DEFAULT_IMAGE_SERVICE: ImageService = "gemini"
 
 export const getImageService = (): ImageService => {
   const variant = getFeatureVariant("image-service")
-  const requestedImageService = variant?.payload?.value as ImageService
 
-  return requestedImageService || DEFAULT_IMAGE_SERVICE
+  if (!!variant && "payload" in variant && variant.payload && variant.enabled) {
```

Comment:
Checks that: we have the variant, the variant isn't a disabled variant, the variant has a payload and for good measure, that it's enabled. Even though if it has a payload by default it should be enabled. 🤷 Trust no one.